### ISSUE TITLE
Add ignore flag for PR

### ISSUE THREAD

**Issue by elmasse** [geut/chaas#14]
We need a way to ignore checks for certain PRs. We should discuss alternatives:

- Flag in PR name: `[ignore-chaas] This is PR Title`. Configurable?
- Flag for commits/PR (chore, docs). Configurable ?
- Use a label in the PR. Configurable

**Comment by dpaez** [comment]
🆒 idea Max.

I was thinking about this, essentially I'm in for all of those but with some good defaults. I.e.: `chaas` is enabled/on by default and you can exclude a PR if you want to, but what can we do to omit certain PRs beforehand. Like having a way to check if it is a docs update PR, skip it. How much does it cost such a job/check? Here I'm thinking about DX. It's like if we have a way to categorize/list a subtype of PRs which shouldn't be checked (skipped by `chaas`, without any config). 

All in all, my vote is for all of the above options in this order maybe:
1. title flag / flag for commits ( <-- do you have any example for a flag similar like this -commits-)
2. label flag (label flag sounds cool but I'm afraid that maybe people forgot to add them)

**Comment by elmasse** [comment]
@dpaez 

Yeah, I'm not quite sold on using custom labels but it's a simple way to make sure you can omit a check. 

On the title flag, the CIs already do this, if you add a `[ignore-ci]` flag in your commits CI will not run. This is my main motivation. But there might be other scenarios too. We currently skip checks if ignore is specified and a PR contains only ignored files. But we don't have a way to ignore at all the check by forcing it.

**Comment by dpaez** [comment]
> On the title flag, the CIs already do this, if you add a `[ignore-ci]` flag in your commits CI will not run. This is my main motivation. But there might be other scenarios too. We currently skip checks if ignore is specified and a PR contains only ignored files. But we don't have a way to ignore at all the check by forcing it.

I didn't know about the `[ignore-ci]` option for CIs. I think it's cool to have an option that is similar/familiar to other systems as well. So we can go ahead with it.
